experimental/PropertyDDS/packages/property-binder/package.json

```diff
@@ -109,8 +110,10 @@
     "outputName": "jest-junit-report.xml"
   },
   "typeValidation": {
+    "disabled": true,
```

Comment:
i see it's called out in the description as having happened, but why are PropertyDDS type tests disabled?

Reply:
We changed the default behavior to enable typetests for all packages in the release group unless disabled. Opt out instead of opt in. The other experimental packages also have type tests disabled.

Many example and test packages don't need them disabled explicitly because they're already excluded through some other means, like being a private package etc.